src/routes/docs/advanced/security/pci/+page.markdoc

```diff
@@ -0,0 +1,12 @@
+---
+layout: article
+title: PCI
+description: Learn about Appwrite's measure to achieve PCI compliance when handling payments and transactions, ensuring secure and safe handling of payment information and personal data.
+---
+The Payment Card Industry Data Security Standard (PCI) is a standard that concerns handling of credit card information, transactions, and payments.
+Appwrite uses [Stripe](https://stripe.com/en-se) to securely handle payments for Appwrite Pro and Scale plans. 
+Stripe is a [PCI Service Provider Level 1](https://www.visa.com/splisting/searchGrsp.do?companyNameCriteria=stripe) provider
+with a strong [commitment to security and privacy](https://stripe.com/docs/security) that matches Appwrite's core values.
+
+If you're looking to add payment or subscription services to your apps built on Appwrite,
+consider using one of the [function templates](https://appwrite.io/docs/products/functions/quick-start).
```

Comment:
`to integrate with other products` as this might hint we recommend to store credit cards on our services and we should be clear this is not the case.